parsl/tests/integration/test_worker_info.py

```diff
@@ -0,0 +1,44 @@
+"""Tests related to Parsl workers being able to access their worker ID"""
```

Comment:
I think this test won't run in pytest in CI because for unpleasant historical parsl reasons, the `/integration/` directory should be ignored - can you move it into a different directory?

Reply:
Any recommendations on a directory to move it to? I was confused about where this would fit. `/sites/`?

Reply:
I've put stuff like this into `/sites/` even though it isn't really a site, so sure.